src/commands/createFunctionApp/containerImage/detectDockerfile.ts

```diff
@@ -20,7 +20,7 @@ export async function detectDockerfile(context: ICreateFunctionAppContext): Prom
         context.rootPath = workspacePath;
 
         //check for host.json location
-        if (!(await isFunctionProject(workspacePath))) {
+        if (await isFunctionProject(workspacePath)) {
             const files = (await findFiles(context.workspaceFolder, `*/${hostFileName}`));
             if (files.length === 0) {
```

Comment:
You honestly probably don't need these checks anymore either.

Reply:
Talked offline and the checks are still needed.